client/js/components/procedure/admin/AdministrationPlaces.vue

```diff
@@ -302,19 +303,17 @@ export default {
     },
 
     /**
-     * When saving a new place the comparison of `foundSimilarName.length === 0` needs to be executed
-     * since it is forbidden to save an already existing place name twice.
-     * When updating a new place the comparison of `foundSimilarName.length === 1` needs to be executed instead
-     * because the current rowData name is already existent in the places object hence updating an existing place
-     * should always be possible.
+     * Checks if the given place name is unique within the list of places.
+     * To update an existing place the currentPlaceId is given and the
+     * place with this id is excluded from the check.
      *
      * @param placeName { string }
-     * @param isNewPlace { boolean }
+     * @param currentPlaceId { string }
      * @returns { boolean }
      */
-    isUniquePlaceName (placeName, isNewPlace = false) {
-      const foundSimilarName = this.places.filter(el => el.name === placeName)
-      return isNewPlace ? foundSimilarName.length === 0 : foundSimilarName.length === 1
+    isUniquePlaceName (placeName, currentPlaceId = '') {
```

Comment:
nitpick: since `name` and `id` both refer to the same place, I think we should use consistent naming (e.g. `currentPlaceName` and `currentPlaceId`)